### ISSUE TITLE
Document "Edit on" links for multiple projects

### ISSUE THREAD

**Issue by benjaoming** [readthedocs/sphinx-multiproject#7]
It's currently complicated to use different repositories for each project and have "Edit on" render correct links. This means being able to add a customizable base URL for each project's repository.

**Comment by stsewd** [comment]
We have this workaround at our docs

https://github.com/readthedocs/readthedocs.org/blob/b2c809b7ed2235c688621abdb396e3cbec697885/docs/conf.py#L152-L157

**Comment by karlsigidex** [comment]
@stsewd yes I found that workaround, I think the point of this ticket though is to:

1. Document the fact that "Edit on" links can become broken when using sphinx-multiproject
2. Document the workaround
3. Document that for the workaround to actually work the user has to request that the RTD admins set the DONT_OVERWRITE_SPHINX_CONTEXT flag

**Comment by benjaoming** [comment]
The workaround sets `conf_py_path` such that the path of `https://github.com/organization/repo/docs/<docset>` is set - but @karlsigidex isn't your use case that you have documentation in several different repositories? Or do you combine all your docs in one repo?

**Comment by karlsigidex** [comment]
@benjaoming we have multiple documents per repository.

In order to better control our documentation builds (e.g. os,python version, packages, etc.) we need to use a .readthedocs.yaml configuration file.

Currently, having a single configuration file at the root of the repository is incompatible with multiple documents per repository. The workaround is to use sphinx-multiproject.

**Comment by benjaoming** [comment]
Gotcha, sounds like the current workaround is applicable :+1:
